pyro/contrib/autoguide/__init__.py

```diff
@@ -519,6 +520,42 @@ def _loc_scale(self, *args, **kwargs):
         return loc, scale
 
 
+class AutoTransformedNormal(AutoLowRankMultivariateNormal):
```

Comment:
You should not inherit from `AutoLowRankMultivariateNormal`, because that class assumes the `._loc_scale()` method produces quantiles of the models variables (not quantiles of an arbitrary latent). Instead I would recommend forking that class's code and defining custom versions of its `.sample_latent()` and `._loc_scale()` methods so that `.median()` and `.quantiles()` will be defined correctly.